unicorn/templates/default/nginx_unicorn_web_app.erb

```diff
@@ -48,9 +48,15 @@ server {
     proxy_redirect off;
 
     <% if @application[:force_ssl] %>
+    set $https_forward "https://$host$uri";
+    # Don't disclose private IP addresses when no host specified (HTTP/1.0)
+    if ($host ~ "\d+\.\d+\.\d+\.\d+") {
+      set $https_forward "https://www.<%= @application[:domains].first %>.com$uri";
```

Comment:
You're right. We don't have our domains configured quite right. The .com shouldn't be there. Will send up a fix.